app/services/early_career_teachers/create.rb

```diff
@@ -33,10 +33,11 @@ def call
 
         ParticipantProfileState.create!(participant_profile: profile, cpd_lead_provider: school_cohort&.default_induction_programme&.lead_provider&.cpd_lead_provider)
         if school_cohort.default_induction_programme.present?
-          Induction::Enrol.call(participant_profile: profile,
-                                induction_programme: school_cohort.default_induction_programme,
-                                mentor_profile:,
-                                start_date:)
+          induction_record = Induction::Enrol.call(participant_profile: profile,
+                                                   induction_programme: school_cohort.default_induction_programme,
+                                                   mentor_profile:,
+                                                   start_date:)
+          induction_record.update!({ appropriate_body_id: })
```

Comment:
Could we change `Induction::Enrol` to take this as an optional param so that we can keep the appearance that induction records are "immutable" ?